src/bindings/python/src/pyopenvino/utils/utils.cpp

```diff
@@ -108,6 +119,9 @@ py::object from_ov_any(const ov::Any& any) {
             PyDict_SetItemString(dict, property_name.c_str(), PyUnicode_FromString(mutability.c_str()));
         }
         return py::cast<py::object>(dict);
+    } else if (any.is<std::shared_ptr<ov::Meta>>()) {
+        const ov::AnyMap& as_map = *any.as<std::shared_ptr<ov::Meta>>();
+        return from_ov_any_map(as_map);
     } else if (any.is<ov::element::Type>()) {
```

Comment:
```suggestion
    } else if (any.is<ov::Meta>()) {
        // impl...
    } else if (any.is<std::shared_ptr<ov::Meta>>()) {
        const ov::AnyMap& as_map = *any.as<std::shared_ptr<ov::Meta>>();
        return from_ov_any_map(as_map);
    } else if (any.is<ov::element::Type>()) {
```
Is `ov::Meta` without shared pointer a case as well?

Reply:
it doesn't make sense as `ov::Meta` is interface class with pure virtual methods.